packages/extras/src/lib/components/PerfMonitor/PerfMonitor.d.ts

```diff
@@ -0,0 +1,17 @@
+import { SvelteComponent } from 'svelte'
+
+export type PerfMonitorProps = {
+  logsPerSecond?: number // Refresh rate of the logs [default 10]
```

Comment:
If we convert these comments to jsdoc they should show up in code editors:
```suggestion
  /**
   * Refresh rate of the logs
   * @default 10
   */
  logsPerSecond?: number
```